src/main/protobuf/soc/board.proto

```diff
@@ -0,0 +1,67 @@
+syntax = "proto2";
+
+package protos.soc;
+
+message Hex {
+    enum Resource {
+        BRICK = 0;
+        ORE = 1;
+        SHEEP = 2;
+        WHEAT = 3;
+        WOOD = 4;
+        DESERT = 5;
+    }
+    required string id = 1;
+    required Resource resource = 2;
+    optional int32 number = 3;
```

Comment:
What is number used for?

Reply:
roll I will rename

Reply:
Oh, I was just being stupid, I think number is probably fine. That said, you might consider:

`message HexProbability { 
    int32 dots = 3
    int32 number = 2
}`

And then `Hex`
```
int32 id = 1
Resource resource = 2
HexProbability probability = 3
```

I honestly don't care either way, just throwing out ideas.